src/components/avatar.tsx

```diff
@@ -0,0 +1,31 @@
+import Image from "next/image";
+
+interface AvatarProps {
+  src?: string;
+  name?: string;
+  role?: string;
+}
+
+const Avatar = ({ src, name, role }: AvatarProps) => {
+  return (
+    <div className="flex flex-shrink-0 items-center gap-4 md:flex-col">
+      <Image
+        src={src ?? "/images/none.png"}
+        alt="Profile picture"
```

Comment:
I think the name of the person in the photo would be a better alternate text.

```suggestion
        alt={name}
```